Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -83,6 +95,68 @@ public class RemoteImageLoader : MonoBehaviour {
 
             completion(www.texture, null);
         }
+
+        /// <summary>
+        /// Loads an image from the provided URL if we don't have one in the cache or if it's new. Results are cached
+        /// so subsequent images are not downloaded from the web.
+        /// </summary>
+        /// <param name="url">URL to fetch image from.</param>
+        /// <param name="completion">Callback invoked with the cached or downloaded texture.</param>
+        /// <returns></returns>
```

Comment:
imo you dont need docs on an implementation detail, its not a big deal tho... Up to you!